plugins/builds/update.js

```
@@ -254,6 +254,22 @@ module.exports = () => ({
                                 // Don't further trigger pipeline if intented to skip further jobs
 
                                 if (newBuild.status !== 'SUCCESS' || skipFurther) {
+                                    // Check for failed jobs and remove any child jobs in created state
+                                    if (newBuild.status === 'FAILURE') {
+                                        try {
+                                            await removeJoinChild(
+                                                {
+                                                    pipeline,
+                                                    job,
+                                                    build: newBuild
+                                                },
+                                                request.server.app
+                                            );
+                                        } catch (err) {
+                                            throw boom.internal(`failed to remove created child jobs, ${err}`);
```

should this throw err or just fail silently?